src/VisualStudio/Core/Def/Implementation/Remote/ServiceHubRemoteHostClient.cs

```diff
@@ -43,9 +40,6 @@ private enum GlobalNotificationState
         private readonly object _globalNotificationsGate = new object();
         private Task<GlobalNotificationState> _globalNotificationsTask = Task.FromResult(GlobalNotificationState.NotStarted);
 
-        private readonly object _currentRemoteWorkspaceNotificationTaskGate = new object();
-        private Task _currentRemoteWorkspaceNotificationTask = Task.CompletedTask;
```

Comment:
no need for a complex task queue to remote messages to oop anymore.

Reply:
Also, if we ever do need queuing here, there are better ways than this to add it back 👍